### PR TITLE
scenario manager rest apis and protobuf message definition

### PR DESCRIPTION
**Pull request by cj-chung**

### CODE REVIEW

**Comment by cj-chung** [comment]
@phudtran, @zzengcs and @yanmo96 Please review this PR, especially for the protobuf definition for your component. I made couple of changes in each protobuf file. If you're agree the contracts, I will merge them into master first.

File: api/proto/v1/network.proto [hunk 1]
```diff
@@ -52,28 +52,6 @@ message InternalNetworkInfo {
     repeated InternalSecurityGroupInfo security_groups = 10;
 }
 
-message InternalServiceInfo {
```
Comment (yanmo96) [inline api/proto/v1/network.proto:55]: Hi @cj-chung, the InternalServiceInfo and InternalComputeInfo may still be needed for the Merak Network component

Comment (cj-chung) [reply api/proto/v1/network.proto:55]: @cj-chung I've move them to `common.proto`, since compute also need the message definition.

Comment (yanmo96) [reply api/proto/v1/network.proto:55]: sounds good, thanks!

File: api/proto/v1/common.proto [hunk 2]
```diff
@@ -23,3 +23,37 @@ message ReturnMessage {
...
+
+message InternalSubnetInfo {
+  string subnet_id = 1;
+  string subnet_cidr = 2;
+  string subnet_gw = 3;
+  uint32 number_vms = 4;
+}
+
+message InternalVpcInfo {
+  string vpc_id = 1;
```
Comment (yanmo96) [inline api/proto/v1/common.proto:57]: We need `vpc_cider` and maybe `vpc_gw` if we want to specific our own

Comment (cj-chung) [reply api/proto/v1/common.proto:57]: @yanmo96 does alcor require vpc_cidr and vpc_gw? In what scenario we need these two parameters?

**Review by zzengcs — APPROVED** [review]
Topology.proto

**Pull request merged**